lang/de.xml

```diff
@@ -25,10 +25,10 @@
 	</main_menubar>
 
 	<about name="Über">
-		<vita3k>Vita3K: a PS Vita/PSTV Emulator. The world's first functional PS Vita/PSTV emulator.</vita3k>
+		<vita3k>Vita3K: a PS Vita/PS TV Emulator. The world's first functional PS Vita/PS TV emulator.</vita3k>
```

Comment:
no sur for this ? 🤔  i have all time thinks is PSTV like PSVita

Reply:
but in Asia it called "PS Vita TV", USA and EU are called "PS TV", so i think it is name different but they same is one thing.

Reply:
oki i see